### PR TITLE
Create Go HTTP server that serves static files

### PR DESCRIPTION
**Pull request by bryk**
@gmarek Can you do the review? I'll do the merge if LGTMed. 

--- 
This is a simple implementaion of a HTTP server serving static files
from CWD. It'll be used for development for a next few weeks.

This can be tested with `# go build console.go` and then `# ./console
--port=8081`.

### CODE REVIEW

File: src/app/backend/console.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+  "flag"
```
Comment (gmarek) [inline src/app/backend/console.go]: In k8s we're using spf13/pflag for flag handling. I'd be nice to be consistent , but it's not a hard requirement.

File: src/app/backend/console.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "fmt"
+  "log"
```
Comment (gmarek) [inline src/app/backend/console.go]: In the same spirit as above: we're using golang/glog for logging.

File: src/app/backend/console.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+func main() {
+  flag.Parse()
+  log.Print("Starting HTTP server on port ", *argPort)
+
+  // Run a HTTP server that serves static files from current directory.
+  // TODO(bryk): Disable directory listing.
+  http.Handle("/", http.FileServer(http.Dir("./")))
+  err := http.ListenAndServe(fmt.Sprintf(":%d", *argPort), nil)
```
Comment (gmarek) [inline src/app/backend/console.go]: We're using glog.Fatal(http.ListenAndServe(...))

**Comment by gmarek** [comment]
Few nits, which you can change to be more consistent with 'main' K8s style (which I think is a good idea), or ignore if you don't think it'll give any benefit.

Other than that LGTM.

File: src/app/backend/console.go (hunk 1 above)
Comment (bryk) [inline src/app/backend/console.go]: Thanks for pointing this! Done.

File: src/app/backend/console.go (hunk 2 above)
Comment (bryk) [inline src/app/backend/console.go]: Done.

File: src/app/backend/console.go (hunk 3 above)
Comment (bryk) [inline src/app/backend/console.go]: Done.

**Pull request merged**